app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -233,24 +329,42 @@ class ProjectInformationActivity : AppCompatActivity() {
         }
     }
 
+    private fun handleVideoWithFavoritePersistenceFiltering(videosLinks: List<String>){
+        userDataDatabase.getListOfFavoriteProjects({ favorites ->
+            addVideoAfterDownloadedWithFavoritePersistence(videosLinks, favorites.contains(projectVar.id))
+        },{
+
+        })
+    }
```

Comment:
Function name starts to become a little too long.

Reply:
I will address this when I do the PR on the video storing class